src/qt/transactionview.cpp

```diff
@@ -414,9 +414,8 @@ void TransactionView::abandonTx()
     QModelIndexList selection = transactionView->selectionModel()->selectedRows(0);
 
     // get the hash from the TxHashRole (QVariant / QString)
-    uint256 hash;
     QString hashQStr = selection.at(0).data(TransactionTableModel::TxHashRole).toString();
-    hash.SetHexDeprecated(hashQStr.toStdString());
+    Txid hash = Txid::FromHex(hashQStr.toStdString()).value();
```

Comment:
Seems fine, same reasoning about `.data()` as before. Plus if we get here, the context menu was enabled.